server/tests/helpers/core-utils.ts

```diff
@@ -148,3 +148,54 @@ describe('Bitrate', function () {
     }
   })
 })
+
+describe('Parse semantic version string', function () {
+
+  it('Should parse Node.js version string', function () {
+    const actual = parseSemVersion('v18.16.0')
+
+    expect(actual.major).to.equal(18)
+    expect(actual.minor).to.equal(16)
+    expect(actual.patch).to.equal(0)
+  })
+
+  it('Should parse FFmpeg version string from Debian 12 repo', function () {
+    const actual = parseSemVersion('5.1.3-1')
+
+    expect(actual.major).to.equal(5)
+    expect(actual.minor).to.equal(1)
+    expect(actual.patch).to.equal(3)
+  })
+
+  it('Should parse FFmpeg version string from Arch repo', function () {
+    const actual = parseSemVersion('n6.0')
+
+    expect(actual.major).to.equal(6)
+    expect(actual.minor).to.equal(0)
+    expect(actual.patch).to.equal(0)
+  })
+
+  it('Should parse FFmpeg version from GitHub release', function () {
+    const actual = parseSemVersion('5.1.3')
+
+    expect(actual.major).to.equal(5)
+    expect(actual.minor).to.equal(1)
+    expect(actual.patch).to.equal(3)
+  })
+
+  it('Should parse FFmpeg version from GitHub dev release', function () {
+    const actual = parseSemVersion('5.1.git')
```

Comment:
Just a small detail, I actually compiled and installed [n6.1-dev](https://github.com/FFmpeg/FFmpeg/releases/tag/n6.1-dev), which showed this version string.
I'm guessing it was just a typo. :thinking:

Goes to show version strings aren't too reliable either. ^-^'